modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/appserver/integration/common/utils/ASIntegrationConstants.java

```diff
@@ -20,4 +20,8 @@
 public class ASIntegrationConstants {
     public static final String JAGGERY_APPLICATION = "jaggery";
     public static final String WEB_APPLICATION = "webapps";
+    public static final String DB_URL  = "//databases/database[@name='%s']/url";
```

Comment:
Rename all variables referring xpaths of automation.xml  ie - CONTEXT_XPATH_DB_CONNECTION_URL